README.md

```diff
@@ -7,11 +7,11 @@ using some simple formatting conventions. You can also author codelabs using mar
 This repo contains all the tools and documentation you’ll need for building and publishing
 your own codelabs.
 
-If you're interested in authoring codelabs, join [this group](https://groups.google.com/forum/#!forum/codelab-authors),
-which connects you with other authors and provides access to the [Codelab Formatting Guide](https://g.co/codelabs/guide).
+If you're interested in authoring codelabs, create a doc following the [Codelab Formatting Guide](FORMAT-GUIDE.md). 
+and see the [claat](claat) directory for a detailed description of the `claat` command line tool.
```

Comment:
We could point to the example doc that you have created here.

Reply:
You mean the template doc? The formatting guides points to it but I can add a ref here too if you think that's useful.

Reply:
Yea I just saw you already pointed to it somewhere else. that's fine then :)

Reply:
Cool, can I get an approval when you get a chance?

Reply:
Thanks for the missing protocols!